authentik/core/migrations/0052_groupancestor_groupparentagenode_group_parents.py

```diff
@@ -0,0 +1,143 @@
+# Generated by Django 5.1.12 on 2025-09-12 08:38
+
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+
+from django.apps.registry import Apps
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+
+
+def migrate_parents(apps: Apps, schema_editor: BaseDatabaseSchemaEditor):
+    Group = apps.get_model("authentik_core", "Group")
+    db_alias = schema_editor.connection.alias
+
+    for group in Group.objects.using(db_alias).all():
+        if not group.parent:
+            continue
+        group.parents.add(group.parent)
+        group.save()
+
+
+create_transitive_closure = """
+CREATE MATERIALIZED VIEW authentik_core_groupancestry AS
```

Comment:
Might want to check https://django-postgres-extra.readthedocs.io/en/master/api_reference.html?highlight=view#psqlextra.models.PostgresMaterializedViewModel